downstream/modules/platform/ref-controller-system-requirements.adoc

```diff
@@ -7,68 +7,39 @@ In the installer, four node types are provided as abstractions to help you desig
+
 For further information about required RAM and CPU levels, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_administration_guide/assembly-controller-improving-performance[Performance tuning for automation controller].
 ====
 
-.Execution nodes
-
-[cols="a,a",options="header"]
-|===
-h| Requirement | Minimum required
-| *RAM* | 16 GB
-| *CPUs* | 4
-| *Local disk* | 40GB minimum
-|===
-
 *Control nodes*
 
 Control nodes process events and run cluster jobs including project updates and cleanup jobs. Increasing CPU and memory can help with job event processing.
 
-.Control nodes
+Control nodes have the following storage requirements:
 
-[cols="a,a",options="header"]
-|===
-h| Requirement | Minimum required
-| *RAM* | 16 GB
-| *CPUs* | 4
-| *Local disk* a|
 * 40GB minimum with at least 20GB available under /var/lib/awx
 * Storage volume must be rated for a minimum baseline of 1500 IOPS
 * Projects are stored on control and hybrid nodes, and for the duration of jobs, are also stored on execution nodes. If the cluster has many large projects, consider doubling the GB in /var/lib/awx/projects, to avoid disk space errors.
-|===
 
 *Hop nodes*
 
 Hop nodes serve to route traffic from one part of the {AutomationMesh} to another (for example, a hop node could be a bastion host into another network). RAM can affect throughput, CPU activity is low. Network bandwidth and latency are generally a more important factor than either RAM or CPU.
 
-.Hop nodes
-
-[cols="a,a",options="header"]
-|===
-h| Requirement | Minimum required
-| *RAM* | 16 GB
-| *CPUs* | 4
-| *Local disk* | 40 GB
-|===
-
-* Actual RAM requirements vary based on how many hosts {ControllerName} will manage simultaneously (which is controlled by the `forks` parameter in the job template or the system `ansible.cfg` file).
-To avoid possible resource conflicts, Ansible recommends 1 GB of memory per 10 forks and 2 GB reservation for {ControllerName}. For more information, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_user_guide/controller-jobs#controller-capacity-determination[{ControllerNameStart} capacity determination and job impact]. If `forks` is set to 400, 42 GB of memory is recommended.
+* Actual RAM requirements vary based on how many hosts {ControllerName} manages simultaneously (which is controlled by the `forks` parameter in the job template or the system `ansible.cfg` file).
+To avoid possible resource conflicts, Ansible recommends 1 GB of memory per 10 forks and 2 GB reservation for {ControllerName}, see link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_user_guide/controller-jobs#controller-capacity-determination[{ControllerNameStart} capacity determination and job impact]. 
```

Comment:
```suggestion
To avoid possible resource conflicts, Ansible recommends 1 GB of memory per 10 forks and 2 GB reservation for {ControllerName}.
See link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/automation_controller_user_guide/controller-jobs#controller-capacity-determination[{ControllerNameStart} capacity determination and job impact]. 
```